plugins/search-backend/src/service/router.ts

```diff
@@ -62,6 +62,7 @@ export type RouterOptions = {
   logger: Logger;
 };
 
+const maxPageLimit = 100;
```

Comment:
What was the thinking on this limit?

Reply:
we looked at similar limits for other apis, stack overflow as one example. Do you have any other recommendations?

Reply:
Good thinking!  I had only looked briefly at the default setting for ES's max page size (which was something like `10000`), but given limitations associated with permissions-enabled searches, it's probably safer at a lower value.